CHANGELOG.md

```diff
@@ -0,0 +1,33 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+## [Unreleased]
+### Added
+- Support of UNIX domain sockets
+- Solaris/illumos support
+- the "reserve-job" command 
+- draining status to the "stats" command
+
+### Changed
```

Comment:
@ysmolsky we should probably add the default sync interval change here too.

Reply:
added!